config/client.go

```diff
@@ -89,6 +90,9 @@ type TiKVClient struct {
 	// TTLRefreshedTxnSize controls whether a transaction should update its TTL or not.
 	TTLRefreshedTxnSize      int64  `toml:"ttl-refreshed-txn-size" json:"ttl-refreshed-txn-size"`
 	ResolveLockLiteThreshold uint64 `toml:"resolve-lock-lite-threshold" json:"resolve-lock-lite-threshold"`
+	// MaxConcurrencyRequestLimit is the max concurrency number of request to be sent the tikv
+	// 0 means auto adjust by feedback.
+	MaxConcurrencyRequestLimit int64 `toml:"max-concurrency-request-limit" json:"max-concurrency-request-limit"`
```

Comment:
This would be a public tidb configuration, should it be approved by the PM member according to the current process requirements?